src/caselawclient/xquery/break_judgment_checkout.xqy

```diff
@@ -0,0 +1,7 @@
+xquery version "1.0-ml";
+import module namespace dls = "http://marklogic.com/xdmp/dls" at "/MarkLogic/dls.xqy";
+
+declare variable $uri as xs:string external;
+let $deep := fn:true()
+
+return dls:break-checkout($uri, $deep)
```

Comment:
> deep
>  If set to true, then the checkout is broken for any checked-out documents that are directly or indirectly included by the specified document.

Seems like a potentially wide-ranging unlock by default, but I don't think we have included documents yet, and it'd be a pain if something wasn't unlocked when we wanted it to be.